features/cellular/framework/targets/QUECTEL/BG96/QUECTEL_BG96.cpp

```diff
@@ -30,6 +30,8 @@ using namespace events;
 
 #define DEVICE_READY_URC "CPIN:"
 
+#define tr_debug(...)
```

Comment:
Please remove this and include the correct header:
#include "CellularLog.h"

Reply:
Just curious, for what purpose CellularLog is? can’t mbed_trace include be used directly here?

Reply:
It defines trace group. There was other reasons in older mbed-os versions.